--- Makefile.orig	Wed Feb 15 10:30:42 2006
+++ Makefile	Thu Feb 16 19:37:37 2006
@@ -66,7 +66,10 @@
 OBJS+= audio.o
 endif
 
-EXTRALIBS += -L../libavutil -lavutil$(BUILDSUF) -lavcodec$(BUILDSUF) -L../libavcodec
+EXTRALIBS += -L../libavutil -lavutil -lavcodec -L../libavcodec
+ifneq ($(CONFIG_WIN32),yes)
+LDFLAGS+=-Xlinker -rpath "$(prefix)/lib/$(PROGNAME)/codecs/"
+endif
 
 ifeq ($(CONFIG_AUDIO_BEOS),yes)
 CPPOBJS+= beosaudio.o
