--- Makefile.orig	Mon Feb 13 12:15:35 2006
+++ Makefile	Mon Feb 13 18:06:02 2006
@@ -51,18 +51,9 @@
 	$(AR) rc $@ $(OBJS)
 	$(RANLIB) $@
 
-
-install-lib-shared: $(SLIBNAME)
-ifeq ($(CONFIG_WIN32),yes)
-	install $(INSTALLSTRIP) -m 755 $(SLIBNAME) "$(prefix)"
-else
-	install $(INSTALLSTRIP) -m 755 $(SLIBNAME) \
-		$(libdir)/$(SLIBNAME_WITH_VERSION)
-	ln -sf $(SLIBNAME_WITH_VERSION) \
-		$(libdir)/$(SLIBNAME_WITH_MAJOR)
-	ln -sf $(SLIBNAME_WITH_VERSION) \
-		$(libdir)/$(SLIBNAME)
-endif
+install: all
+	install -d  $(prefix)/lib/$(PROGNAME)/codecs/
+	install $(INSTALLSTRIP) -m 755 $(SLIBNAME) $(prefix)/lib/$(PROGNAME)/codecs/$(SLIBNAME)
 
 install-lib-static: $(LIB)
 	install -m 644 $(LIB) "$(libdir)"
@@ -72,7 +63,7 @@
 	install -m 644 postprocess.h "$(incdir)/postprocess.h"
 	install -m 644 $(SRC_PATH)/libpostproc.pc "$(libdir)/pkgconfig"
 
-
+uninstall:
 #
 # include dependency files if they exist
 #
